simulation/test_eor_sim.pro

```diff
@@ -7,19 +7,23 @@ FUNCTION test_eor_sim
 
 ; Visibility degrid requires -- obs, psf, image_uv, and vis_weight
 
-instrument='paper'
+instrument='mwa'
```

Comment:
Please move this file to the pipeline_scripts/FHD_IDL_wrappers repo

Reply:
I've opened a PR on pipeline_scripts and removed the file from here.